langchain/prompts/prompt.py

```diff
@@ -97,3 +97,14 @@ def from_examples(
         example_str = example_separator.join(examples)
         template = prefix + example_str + suffix
         return cls(input_variables=input_variables, template=template)
+
+    @classmethod
+    def from_file(cls, template_file: str, input_variables: List[str]) -> "Prompt":
+        """Load a prompt from a file.
+
```

Comment:
can we add an Args section (to be consistent with other docstrings)

Reply:
Yes, I've update it with the Args